api/endpoint/rates.go

```diff
@@ -42,3 +42,22 @@ func GetRate(controller controllers.RatesController) func(c *gin.Context) {
 		c.JSON(http.StatusOK, response)
 	}
 }
+
+// @Summary Get Fiat Rates
+// @Description Get Fiat Rates
+// @Accept json
+// @Produce json
+// @Tags Rates
+// @Success 200 {object} controllers.FiatRates
+// @Router /fiat_rates [get]
```

Comment:
I would suggest to use `v1/fiat_rates` for consistency

Reply:
https://github.com/trustwallet/watchmarket/commit/1fa2a2dddff9b6e248de0f5084d334fed541d70c